packages/venia-ui/lib/components/OrderHistoryPage/orderHistoryPage.js

```diff
@@ -115,6 +118,28 @@ const OrderHistoryPage = props => {
         />
     );
 
+    const pageInfoLabel = pageInfo ? (
+        <FormattedMessage
+            defaultMessage={'Showing {current} of {total}'}
+            id={'orderHistoryPage.pageInfo'}
+            values={pageInfo}
+        />
+    ) : null;
+
+    const loadMoreButton = loadMoreOrders ? (
```

Comment:
Interesting. I was expecting to see a flag `canLoadMoreOrders` or something but this function will be `null` if you can't load more. I'm okay with this, I like that it allows for fewer props between the talon and the component and that we don't have to define a function that we know won't be called.